docker-compose-generator/docker-fragments/opt-add-thunderhub.yml

```diff
@@ -2,11 +2,11 @@ version: "3"
 services:
     btcpayserver:
         environment:
-            BTCPAY_BTCEXTERNALTHUNDERHUB: "server=/thub/;cookiefile=/etc/lnd_bitcoin_thub_datadir/.cookie"
+            BTCPAY_BTCEXTERNALTHUNDERHUB: "server=/thub/sso/;cookiefile=/etc/lnd_bitcoin_thub_datadir/.cookie"
```

Comment:
@NicolasDorier had tested this part as well and it had correctly gone to `/thub/sso?token=...` but just updated and it's redirected to `/sso?token=...` any idea why?

Reply:
Fixed here: https://github.com/btcpayserver/btcpayserver-docker/pull/405